protocol/types.go

```diff
@@ -187,3 +189,34 @@ type VersionResponse struct {
 	// Build contains the timestamp at the time of the build.
 	Build time.Time `json:"build"`
 }
+
+// SupportedLanguagesRequest is a request to get the supported languages
+//proteus:generate
+type SupportedLanguagesRequest struct{}
+
+// SupportedLanguagesResponse is the reply to SupportedLanguagesRequest
+//proteus:generate
+type SupportedLanguagesResponse struct {
+	Response
+	// Drivers are the loaded drivers
+	Drivers []DriverDetails `json:"drivers"`
+}
+
+// DriverDetails is the installed driver exported data
+//proteus:generate
+type DriverDetails struct {
+	Name     string `json:"name"`
+	Language string `json:"language"`
+	Version  string `json:"version"`
+	Status   string `json:"status"`
```

Comment:
Could also have the `features` array of the manifest that gives a list of supported features.

Reply:
features it's not the languages supported but the level of implementation (ast, uast, roles, and in the close future high level objects).

Reply:
I just don't see a reason why we should abstract away the real manifest that we have for the driver (this is what suggested by `SupportedLanguages` and omitting `features` field). A user can drop any unnecessary fields if he likes, but we won't be able to get missing fields if we decide to not include them.

Reply:
So you both @juanjux and @dennwc see a benefit if we include `features` field in the exposed data regardless the name of the method used?
If you agree I'll add it for sure :dagger:

Reply:
@juanjux I added `features` to the exposed data